frb_rust/Cargo.toml

```diff
@@ -78,5 +72,8 @@ chrono = ["dep:chrono", "allo-isolate/chrono"]
 wasm-start = ["console_error_panic_hook"]
 thread-pool = ["dep:threadpool"]
 rust-async = ["dep:tokio", "dep:futures", "dep:wasm-bindgen-futures"]
-user-utils = ["dep:android_logger", "dep:oslog"]
+user-utils = []
```

Comment:
design choice question: shall we by default log to Flutter, or log to android/ios system log? If the former, if our Flutter somehow fails, then we cannot log things visibly.

Reply:
AFAIK (from trying myself) the stdout is logged to the oslog anyways. I did not try it with Android and in iOS with a simulator only.
But I think it is unlikely that Flutter fails (but the iOS/Android logger not) and more user/beginner friendly to log to stdout per default.

I suggest that we improve/reintroduce these mobile loggers if really needed. I will stumble over that in this case anyways.

Reply:
Hmm not sure why I missed this reply. Anyway we have discussed it below, so no worries!